client-app/shared/account/components/sign-in-form.vue

```diff
@@ -145,4 +148,23 @@ const onSubmit = handleSubmit(async () => {
 watch(values, () => {
   error.value = undefined;
 });
+
+const errorMessages = computed<{ [key in IdentityErrors]: string }>(() => ({
+  [IdentityErrors.LOGIN_FAILED]: t("shared.account.sign_in_form.errors.login_failed"),
+  [IdentityErrors.EMAIL_VERIFICATION_REQUIRED]: t("shared.account.sign_in_form.errors.email_verification_is_required"),
+  [IdentityErrors.USER_IS_TEMPORARY_LOCKED_OUT]: t("shared.account.sign_in_form.errors.user_is_temporary_locked_out"),
+  [IdentityErrors.USER_IS_LOCKED_OUT]: t("shared.account.sign_in_form.errors.user_is_locked_out"),
+  [IdentityErrors.USER_CANNOT_LOGIN_IN_STORE]: t("shared.account.sign_in_form.errors.user_cannot_login_in_store"),
+  [IdentityErrors.PASSWORD_EXPIRED]: t("shared.account.sign_in_form.errors.password_expired"),
+  [IdentityErrors.USER_NOT_FOUND]: t("shared.account.sign_in_form.errors.user_not_found"),
+}));
```

Comment:
I'd like to use the same approach or even the same code, if possible https://github.com/VirtoCommerce/vc-theme-b2b-vue/blob/50d489b7faf1199ea537f045ae0ba657fd75467b/client-app/core/composables/useIdentityErrorTranslator.ts#L7-L10

Reply:
Separate task https://virtocommerce.atlassian.net/browse/ST-5152